### DISCUSSION TITLE
Stuck after moving a Github repo into an organization

### DISCUSSION THREAD

**Discussion by rignaneseleo** [codemagic-ci-cd/codemagic-docs#2186] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

1. Move a github repo that is already on Codemagic into a new organization

Two cases:
2a. Try to update the repo URL from the Codemagic app workflow editor
or
2b. Add the new repo to a new app

### Expected results

2a. Let me change and update the URL
or
2b. Let me create a new app with the new URL and move the settings there

### Actual results

2a. The URL is read only (even though there's a "UPDATE REPO URL" button)
or
2b. The new repo doesn't show up even though I have given permission to access all the repo in the new organization

### Build id (optional)

APP ID: 646148e4d6e708a0b4dac363

**Comment by icarusdust** [comment]
Hi @rignaneseleo, the `Update repository URL` button can be clicked to update the URL after moving your repo to an organization.Testing it right now, and as soon as I move it to my personal account from Github organization or vice versa, I can update the repo URL by clicking the button. 
<img width="973" alt="Screenshot 2023-11-14 at 13 12 19" src="https://github.com/codemagic-ci-cd/codemagic-docs/assets/43378371/a8bb574e-8598-461c-8ec4-c539fa0bb4f9">

Please note that the button is not read-only, and I am checking the app id you provided, it looks alright.

**Comment by rignaneseleo** [comment]
Yeah, somehow the next day it the Repo URL got updated automatically.
